addons/mass_mailing/models/mailing.py

```diff
@@ -442,10 +442,14 @@ def action_view_traces_sent(self):
 
     def _action_view_traces_filtered(self, view_filter):
         action = self.env["ir.actions.actions"]._for_xml_id("mass_mailing.mailing_trace_action")
-        action['name'] = _('%s Traces') % (self.name)
+        action['name'] = _('Sent Mailings')
         action['context'] = {'search_default_mass_mailing_id': self.id,}
         filter_key = 'search_default_filter_%s' % (view_filter)
         action['context'][filter_key] = True
+        action['views'] = [
+            (self.env.ref('mass_mailing.mailing_trace_view_tree_mail').id, 'tree'),
+            (self.env.ref('mass_mailing.mailing_trace_view_form').id, 'form')
```

Comment:
If you want the default form, I don't think you have to specify it (or you could do something like `0, 'form'`)

Reply:
Makes inherit in _sms more explicit in my opinion ... otherwise indeed it is default value. But here I want it to be explicit.